src/common/ceph_strings.cc

```diff
@@ -49,6 +49,22 @@ const char *ceph_osd_state_name(int s)
 	}	
 }
 
+const char *ceph_osd_release_name(int r)
```

Comment:
could `s/int/uint8_t/` to be more consistent. or better off `typedef`/`using` a new type.